CaloFilters/fcl/prolog_trigger.fcl

```diff
@@ -72,12 +72,21 @@ CaloFilters : {
 #skelethon for a cosmic muon calibration
     caloCalibCosmicFilter : {
 	module_type            : CaloCosmicCalib
-	CaloClusterCollection  : "CaloClusterFast"
+	CaloClusterModuleLabel : "CaloClusterFast"
 	debugLevel             : 0	      
-	MinNCrystalHits        : 7	      
-	MinEnergy              : 150.0    #MeV
+	MinNCrystalHits        : 0	      
+	MinEnergy              : 10.0    #MeV
 	MaxEnergy              : 600.0    #MeV
-	triggerPath             : "CaloCalibCosmic"	    
+	triggerPath            : "CaloCalibCosmic"
+	MinNumCelinout         : 6 
+	MinNumCeldiagver       : 6
+	MinCelEneCut           : 15.0   # MeV
+	OutRadius              : 580   # mm
+	InnerRadius            : 460   # mm
+	crystalsize            : 35    # mm (include wrap)
```

Comment:
Since the crystal size comes from the GeometryService isn't this unnecessary?  I think it won't run, as the crystalsize variable has been removed from the module.

Reply:
All comments have been addressed.